Packages.props

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+		<PackageReference Update="Funcky" Version="[1.2, 2)" />
+		<PackageReference Update="Microsoft.AspNetCore.Http.Abstractions" Version="[2.2, 3)" />
+	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Update="Messerli.CodeStyle" Version="1.0.1" />
```

Comment:
Make this a range

Reply:
(Consistent with https://github.com/messerli-informatik-ag/query-provider/pull/10/files)